vipps-recurring-api-faq.md

```diff
@@ -362,16 +362,14 @@ in Vipps FAQs.
-| Invalid payment source (retryable)              | Kortet er ugyldig, prøv å endre det og kontakt `merchant name` å fortsette avtalen. | The card is invalid, try changing it and contacting `merchant name` to continue the agreement. |
-| Invalid payment source (not retryable)          | Kortet er ugyldig, prøv et annet.                                                   | The card is invalid, try another.                                                              |
-| Charge amount too high (variable amount)        | Beløpet er høyere enn det avtalte maksimumsbeløpet.                                 | The amount is higher than the agreed maximum amount.                                           |
-| Future charge amount too high (variable amount) | Du må øke maksbeløpet i avtalen med `merchant name`                                 | You have to change the maximum amount in the agreement. If not, the payment will fail.         |
-
+| Event                                           | Push message text (Norwegian)                                                       | Push message text (English)                                                              |
+|-------------------------------------------------|-------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|
+| Agreement request                               | Bekreft fast betaling                                                               | Confirm recurring payment                                                                |
+| Charge processed successfully                   | Fast betaling til `merchant name` har blitt gjennomført                             | Payment to `merchant name` has been processed                                            |
+| Insufficient funds                              | Betalingen feilet. Pass på at du har nok penger på konto eller bytt kort. Vi prøver igjen senere.                                                                                                                                 | Payment failed. Make sure you have enough money or change card. We'll try again later.   |
+| Invalid payment source                          | Kortet som er knyttet til avtalen din er ugyldig. Bytt til et annet.                | The card associated with your agreement is invalid. Switch to another.                   |
+| Charge amount too high (variable amount)        | Betalingen feilet. Endre maksbeløpet i avtalen under. Vi prøver igjen senere.       | Payment failed. Change the maximum amount in the agreement below. We'll try again later. |
+| Future charge amount too high (variable amount) | Betalingen kommer til å feile. Endre maksbeløpet i avtalen under.                   | Payment will fail. Change the maximum amount in the agreement below.                     |
```

Comment:
These texts are not for notifications. please check the texts in NotificationMessages.kt